src/main/java/core/basesyntax/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -38,11 +41,36 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex,
+            WebRequest request
+    ) {
+        Map<String, Object> body = new LinkedHashMap<>();

```

Comment:
I would suggest to create dedicated class for exception response body (maybe even a record can fit here good).